docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,82 @@
+[role="xpack"]
+[[xpack-logs-analysis-page]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+The *Analysis* page in the Logs app allows you to automatically detect some kinds of log anomalies using machine learning.
+
+The analysis automatically highlights periods where the log rate is outside the expected limits and therefore may be anomalous.
+This helps you to spot suspicious behavior without significant human intervention.
+You can use this information as a basis for further investigations.
+
+On the *Analysis* page, you can inspect the anomalies and the log partitions in which they occurred.
+You can also view the anomalies directly in the Machine Learning app to get a greater understanding of the issues.
+
+The *Analysis* page in the Logs app is only available if you have a license that includes the machine learning features.
```

Comment:
I think this sentence adds value. The Analysis tab is a feature in the **Logs** app which is only available if the **Machine Learning** app is licensed. 

If the user can't see it, I think they need to know why.

Reply:
Because the licensing information is maintained on the https://www.elastic.co/subscriptions page, it is better not to double document this type of content. If for any reason they decide to change what is included in each license and this information becomes stale, it would confuse our users, which is why we don't include licensing content in our docs.

Reply:
Machine learning features exist in multiple licensing levels, but I suspect this combination of Logs + ML functionality is only available in a subset of those levels. I don't think it's appropriate to try to keep those details up-to-date in the documentation; the https://www.elastic.co/subscriptions site is the source of truth.  I've added a suggestion elsewhere in this PR that will hopefully enable you to omit it here.